app/lib/lnd/neutrino.js

```diff
@@ -78,10 +78,20 @@ class Neutrino extends EventEmitter {
     const neutrinoArgs = [
       `--configfile=${this.lndConfig.configPath}`,
       `--lnddir=${this.lndConfig.dataDir}`,
-      `${this.lndConfig.autopilot ? '--autopilot.active' : ''}`,
       `${this.lndConfig.alias ? `--alias=${this.lndConfig.alias}` : ''}`
     ]
 
+    // If the user toggled autopilot on let's enable it,
+    // make sure channels are private and allow it to use 0 conf txs
+    if (this.lndConfig.autopilot) {
+      // Turn autopilot on
+      neutrinoArgs.push('--autopilot.active')
+      // Ensure all channels opened by autopilot will be flagged private
+      neutrinoArgs.push('--autopilot.private')
```

Comment:
should we move these additional options into `resources/lnd.conf`? I think the only thing we need to specify dynamically here is the `--autopilot.active` switch.